demos/smart_classroom_demo_gapi/include/tracker.hpp

```diff
@@ -331,3 +296,15 @@ class Tracker {
 
 int LabelWithMaxFrequencyInTrack(const Track &track, int window_size);
 std::vector<Track> UpdateTrackLabelsToBestAndFilterOutUnknowns(const std::vector<Track>& tracks);
+
+struct TrackerParamsPack {
+    TrackerParams tracker_reid_params;
+    TrackerParams tracker_action_params;
+};
+
+struct TrackersPack {
+    TrackersPack(TrackerParams tracker_reid_params, TrackerParams tracker_action_params)
+        : tracker_reid(Tracker(tracker_reid_params)), tracker_action(Tracker(tracker_action_params)) {}
+    Tracker tracker_reid;
+    Tracker tracker_action;
+};
```

Comment:
This set of structures is also questionable. `TrackersPack` is the state, and `TrackerParamsPack` is its compile args?

Can we have two stateful kernels instead of one?